Hunting Queries/AuditLogs/UserDetectPrivilegeGroup.yaml

```diff
@@ -12,7 +12,7 @@ tactics:
 relevantTechniques:
   - T1548
 query: |
-   let PrivilegedUsers = (_GetWatchlist('Accounts') | project Users);
```

Comment:
Do we have schema of this watchlist?

Reply:
Hi @devikamehra presently we are unable to procure schema of this watchlist, thanks!

Reply:
Can we check with the author on this?

Reply:
We are unable to get in touch with the author currently